src/schemas.ts

```diff
@@ -741,6 +741,12 @@ Given a 3D point [X Y Z]', the projection (x, y) of the point onto the rectified
 This holds for both images of a stereo pair.
 `,
     },
+    {
+      name: "frame_id",
+      type: { type: "primitive", name: "string" },
+      description:
+        "Frame of reference for the camera. The origin of the frame is the optical center of the camera. +x points to the right in the image, +y points down, and +z points into the plane of the image.",
```

Comment:
Is it worth calling out that CameraCalibration should be in the same frame as its corresponding image? What happens if the CameraCalibration is a different frame?

Reply:
No idea really. Probably bad things.